tt_metal/hw/firmware/src/erisc.cc

```diff
@@ -25,6 +25,9 @@ void ApplicationHandler(void) __attribute__((__section__(".init")));
 namespace kernel_profiler {
 uint32_t wIndex __attribute__((used));
 }
+
+tt_l1_ptr mailboxes_t * const mailboxes = (tt_l1_ptr mailboxes_t *)(MEM_MAILBOX_BASE);
```

Comment:
This is a big no no. We can't write to MEM_MAILBOX_BASE (which is 0x4) on the eth core. Can you add a MEM_MAILBOX_BASE to `eth_l1_address_map.h` at `COMMAND_Q_BASE - 204 - 128`.

Reply:
And if you want to help me out here, can you change `range_max = eth_l1_mem::address_map::FIRMWARE_BASE + eth_l1_mem::address_map::ERISC_FIRMWARE_SIZE` in `llrt.cpp` to `range_max = eth_l1_mem::address_map::MEM_MAILBOX_BASE` :))

Reply:
I've updated as requested 👍